### PR TITLE
Keep gem up to date with bundler and ruby versions

### PR DESCRIPTION
**Pull request by alejandrodevs**
## Addresses issue: [#TOOL-16](https://amcoit.atlassian.net/browse/TOOL-16)

Actions:
- Upgrade gem structure to the newest bundler version.
- Upgrade dev dependencies.
- Add latest stable ruby version in travis file.

### CODE REVIEW

**Review by luismg23 — APPROVED** [review]
LGTM :shipit:

File: lib/gamco/builder.rb [hunk 1]
```diff
@@ -34,10 +34,10 @@ def gtag_script(tag_id, options)
...
 
     def gtag_script_content(tag_id, options)
-      "window.dataLayer = window.dataLayer || [];"\
-      "function gtag(){dataLayer.push(arguments);}"\
-      "gtag('js', new Date());"\
-      "gtag('config', '#{tag_id}', #{options.to_json});"
+      "window.dataLayer = window.dataLayer || [];" \
+        "function gtag(){dataLayer.push(arguments);}" \
+        "gtag('js', new Date());" \
+        "gtag('config', '#{tag_id}', #{options.to_json});"
```
Comment (javierg) [inline lib/gamco/builder.rb:40]: why the extra spaces?

**Review by javierg — APPROVED** [review]
:shipit: 

one question about whitespace that seems to be added... not sure if it is just dispaly on GH UI??

File: lib/gamco/builder.rb (hunk 1 above)
Comment (alejandrodevs) [reply lib/gamco/builder.rb:40]: This was corrected by rubocop.

**Pull request merged**